arkane/statmech.py

```diff
@@ -497,7 +521,7 @@ def load(self, pdep=False, plot=False):
                                            multiplicity=conformer.spin_multiplicity)
             else:
                 bond_corrections = 0
-            e_electronic_with_corrections = e_electronic + atom_corrections + bond_corrections
+            e_electronic_with_corrections = e_electronic + atom_corrections + bond_corrections + isodesmic_correction
```

Comment:
This isodesmic correction is at 298K (which implicitly includes the atom thermal corrections).  For an accurate `E0`, I think we should be subtracting the atom thermal corrections here, then add them back to calculate `H298K`.  When I did this, my `H0` (which I assume is E0?) was in better agreement with literature (ATcT).  This is related to PR #1871. If E0 != H0 though, then this isn't necessary.